### PR TITLE
Performance

### PR DESCRIPTION
**Pull request by kcontreras01**

### CODE REVIEW

File: components/AboutPage/index.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const AboutPage = () => {
     text = "For Black women, the fight wasn't over in 1920. Their movement for gender and racial equality continues.",
     graphic = "/static/graphic/about-page/about-graphic-hero.svg";
 
-  const Offlink = ({url, text}) => <a href={url} target={"_blank"}>{text}</a>
+  const Offlink = ({url, text}) => <a href={url} target={"_blank"} rel="noopener">{text}</a>
```
Comment (AudreyAltman) [inline components/AboutPage/index.js:13]: @mdellabitta sanity check - `rel="noopener"` is okay for SEO, right?

**Review by AudreyAltman — APPROVED** [review]

File: components/AboutPage/index.js (hunk 1 above)
Comment (mdellabitta) [reply components/AboutPage/index.js:13]: Yeah, that means the page that opens doesn't have access to `window.opener`... iow, the javascript context of the page that opened it.

File: components/BWSHead/index.js [hunk 2]
```diff
@@ -24,33 +24,43 @@ class BWSHead extends React.Component {
...
+          <meta name='application-name' content='BWS Collection' />
+          <meta name="referrer" content="origin-when-cross-origin" />
 
-          <meta
-            name="viewport"
-            content="width=device-width, initial-scale=1, shrink-to-fit=no"
-          />
+          <meta name='apple-mobile-web-app-capable' content='yes' />
+          <meta name='apple-mobile-web-app-status-bar-style' content='default' />
+          <meta name='apple-mobile-web-app-title' content='BWS Collection' />
```
Comment (mdellabitta) [inline components/BWSHead/index.js:32]: Should we spell out `Black Women's Suffrage`? I think we should if there's room.

File: components/BWSHead/index.js [hunk 3]
```diff
@@ -24,33 +24,43 @@ class BWSHead extends React.Component {
...
           {pageImageCaption &&
-            <meta name="twitter:image:alt" content={pageImageCaption} />}
+            <meta name="twitter:image:alt" content={pageImageCaption} />} 
+          <meta name='twitter:creator' content='@dpla' />
+          <meta name='twitter:site' content='@dpla' />
+          <meta property='og:type' content={seoType || "website"} />
+          <meta property='og:title' content={pageTitle || defaultPageTitle} />
+          <meta property='og:description' content={pageDescription || defaultDescription}/>
+          <meta property='og:site_name' content={defaultPageTitle} />
+          <meta property='og:url' content='https://blackwomenssuffrage.dp.la' />
```
Comment (mdellabitta) [inline components/BWSHead/index.js:62]: Looks like this should be the URL of the page as well: https://neilpatel.com/blog/open-graph-meta-tags/

File: components/BWSHead/index.js [hunk 4]
```diff
@@ -24,33 +24,43 @@ class BWSHead extends React.Component {
...
-          <meta name="referrer" content="origin-when-cross-origin" />
-          <meta
-            name="description"
-            content={pageDescription || defaultDescription}
-          />
-
-          <meta name="og:site_name" content={defaultPageTitle} />
-          <meta name="og:title" content={pageTitle || defaultPageTitle} />
-          <meta name="og:type" content={seoType || "website"} />
-          <meta name="og:image" content={pageImage || defaultImageUrl} />
```
Comment (mdellabitta) [inline components/BWSHead/index.js:42]: Looks like this tag was dropped

File: components/HomePage/HomePage.module.scss [hunk 5]
```diff
@@ -5,7 +5,7 @@
   z-index: 1;
 
   .banner__background {
-    background-image: url("/static/image/home-page/home-image-hero.png");
+    background-image: url("/static/image/home-page/home-image-hero.jpg");
```
Comment (mdellabitta) [inline components/HomePage/HomePage.module.scss:8]: IIRC we had to revert this because the JPG wasn't doing transparency.

**Review by mdellabitta — CHANGES_REQUESTED** [review]

File: components/BWSHead/index.js (hunk 4 above)
Comment (kcontreras01) [reply components/BWSHead/index.js:42]: I can't tell which tag you're referring to. Looks like this comment is in the middle of a block of deleted lines

File: components/BWSHead/index.js [hunk 6]
```diff
@@ -24,33 +24,43 @@ class BWSHead extends React.Component {
...
+                    
+          <link rel='apple-touch-icon' sizes='180x180' href='/static/favicons/apple-touch-icon.png' />
+          <link rel='icon' type='image/png' sizes='32x32' href='/static/favicons/favicon-32x32.png' />
+          <link rel='icon' type='image/png' sizes='16x16' href='/static/favicons/favicon-16x16.png' />
+          <link rel='manifest' href='/static/manifest.json' />
+          <link rel='mask-icon' href='/static/favicons/safari-pinned-tab.svg' color='#5bbad5' />
+          <link rel='shortcut icon' href='/static/favicons/favicon.ico' />
+              
+          <meta name='twitter:card' content='summary' />
+          <meta name='twitter:url' content='https://blackwomenssuffrage.dp.la' />
```
[the remark this answers is not in the document]
Comment (kcontreras01) [reply components/BWSHead/index.js:50]: Should I be passing the current URL as props to the BWSHead component or can I get the route in the BWSHead component?

**Pull request merged**